e2e/service_anvil_test.go

```diff
@@ -0,0 +1,170 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+
+	"github.com/buildwithgrove/path/qos/jsonrpc"
+)
+
+// Anvil JSON-RPC method definitions.
+// These are standard Ethereum methods that require no parameters.
+var anvilExpectedID = jsonrpc.IDFromInt(1)
+
+const (
+	anvilBlockNumber = "eth_blockNumber"
+	anvilChainId     = "eth_chainId"
+	anvilNetVersion  = "net_version"
+)
+
+// getAnvilTestMethods returns Anvil JSON-RPC methods for load testing.
+// All methods use empty parameters for simplicity.
+func getAnvilTestMethods() []string {
+	return []string{
+		anvilBlockNumber,
+		anvilChainId,
+		anvilNetVersion,
+	}
+}
+
+// getAnvilVegetaTargets creates HTTP targets for Anvil load testing.
+// Performs basic health check by fetching current block number.
+func getAnvilVegetaTargets(
+	ts *TestService,
+	methods []string,
+	gatewayURL string,
+) (map[string]vegeta.Target, error) {
+	headers := getRequestHeaders(ts.ServiceID)
+
+	// Verify Anvil is responding and get current block number
+	blockNumber, err := getAnvilBlockNumber(ts, headers, gatewayURL)
```

Comment:
Since `anvil` does not support archival checks I believe we can simplify this by just using the string "latest" here and remove `getAnvilBlockNumber` and all methods called by it.

Reply:
Actually, on another look at the test methods I believe we can remove the assignment to `ts.ServiceParams.blockNumber` altogether since `anvil` uses no methods that care about the block number.